src/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const API_URL = process.env.REACT_APP_API_URL
+export const API_KEY = process.env.REACT_APP_API_KEY
```

Comment:
are you sure its safe to exposing these secrets? I think in future you should create some api proxy server to be sure that no one would steal it

Reply:
The problem is that the API key is attached to every api call as query param anyway so I can't really hide it